CMakeLists.txt

```diff
@@ -67,6 +67,13 @@ add_definitions(
         -DVCS_REVISION_ID=0x${vcs_revision_id}ULL
         -DNODE_NAME="org.opencyphal.ocvsmd"
 )
+if (DEFINED PLATFORM_LINUX_TYPE)
+    if(${PLATFORM_LINUX_TYPE} STREQUAL "bsd")
+        add_definitions(-DPLATFORM_LINUX_TYPE_BSD)
```

Comment:
BSD is not Linux, it's a different kernel family, so `PLATFORM_LINUX_TYPE` may need renaming.